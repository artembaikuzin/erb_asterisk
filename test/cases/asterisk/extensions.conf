#include "extensions.conf.includes"

[inbound]
; Yield for :global_inbound
; priority: 10
include => office-inbound-prior
include => office-inbound


exten => v[N]oV[!][.]123,1,NoOp
same => Dial(IAX2/longdistance)
same => Hangup

exten => _Lo[n]gE[x]te[n]sio[n]1234[!]-X.,1,NoOp

[outbound]
; Yield for :outbound
; priority: 333

exten => 1234,1,NoOp


exten => _super_e[x]te[n]sio[n]-X.,1,NoOp
same => n,Goto(another-extension,${EXTEN:16},1)


