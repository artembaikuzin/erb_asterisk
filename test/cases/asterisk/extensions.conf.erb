#include "extensions.conf.includes"

[inbound]
<%= yield_here :global_inbound %>

exten => <%= escape_exten 'vNoV!.123' %>,1,NoOp
same => Dial(IAX2/longdistance)
same => Hangup

exten => _<%= escape_exten 'LongExtension1234!' %>-X.,1,NoOp

