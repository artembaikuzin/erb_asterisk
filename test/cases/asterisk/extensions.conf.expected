#include "extensions.conf.includes"

[inbound]
; Yield for :global_inbound
include => office-inbound

exten => v[N]oV[!][.]123,1,NoOp
same => Dial(IAX2/longdistance)
same => Hangup

exten => _Lo[n]gE[x]te[n]sio[n]1234[!]-X.,1,NoOp

